src/main/groovy/life/qbic/datamodel/accounting/ProductItem.groovy

```diff
@@ -45,11 +45,11 @@ class ProductItem {
     }
 
     /**
-     * Sums up the cost of the item based on its quantity and the price and unit defined in the product
+     * Calculates the cost of the item based on its quantity and the price and unit defined in the product
      * @return
      */
     double computeTotalCosts(){
-        //todo implement me
+        return product.unitPrice * quantity
```

Comment:
Are overheads also included here? Or is this not the right place to put this?

Reply:
I was not sure about it. I just put the implementation that was obvious for me. We can still change it when we realize we need another calculation here, right?

Reply:
Sure. We just need to keep this in mind. It is also the most intuitive way to calculate the price of a product item.

Reply:
Right, we also need to think about where we want to place more specific calculations. Could also imagine that overheads and stuff could be added directly in the use case